practical-07/src/components/LoginForm.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+<div class="p-4">
+  <div class="card  align-items-center">
+        <Form @submit="handleSubmit" :validation-schema="schema" class="p-6 row col-sm-4">
+            <div class="heading">Log In</div>
+ 
+            <div class="form-group mb-2">
+                <label for="email">Email address :</label>
+                <Field id="email" name="email" type="email" class="form-control" placeholder="Enter E-mail" />
+                <ErrorMessage name="email" class="text-danger" />
+            </div>
+
+            <div class="form-group mb-2">
+                <label for="password">Password :</label>
+                <Field id="password" name="password" type="password" class="form-control" placeholder="Enter Password" />
+                <ErrorMessage name="password" class="text-danger" />
+            </div>
+
+             <div class="col-auto">
+                <button type="submit" class="btn btn-success">Log In</button>
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
          <button type="submit" class="btn btn-success">Log In</button>
```